tests/db/test_v202111260_to_v202203160.py

```diff
@@ -0,0 +1,113 @@
+"""
+    Copyright 2022 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+import os
+from typing import AsyncIterator, Awaitable, Callable, List
+
+import pytest
+from asyncpg import Connection
+
+from db.common import PGRestore
+from inmanta.const import ResourceState
+from inmanta.data import Resource
+from inmanta.server.bootloader import InmantaBootloader
+
+
+@pytest.fixture
+async def migrate_v202111260_to_v202203160(
+    hard_clean_db, hard_clean_db_post, postgresql_client: Connection, server_config
+) -> AsyncIterator[Callable[[], Awaitable[None]]]:
```

Comment:
Did you know there is now the `migrate_db_from` fixture defined in `conftest.py`?